app/views/passengers/index.haml

```diff
@@ -46,9 +46,9 @@
           %td= passenger.email
           %td= passenger.phone
           - unless @filters.include? 'Active'
-            %td.yes= yes_no_image passenger.active?
-          %td.yes= yes_no_image passenger.permanent?
-          %td.yes{data: {sort: sortable_date(passenger.doctors_note)}}=passenger.expiration_display
```

Comment:
Whoops. You _do_ want to keep the `data-sort` attribute, though.

Reply:
woops, `dt=` was pressed.